examples/cloudflare-workers/server/index.ts

```diff
@@ -4,10 +4,14 @@ import { Hono } from 'hono'
 const app = new Hono<{
   Bindings: {
     MY_VAR: string
-  }
+  },
+  Variables: {
+    'hono-context': string
+  },
 }>()
 
 app.use(async(c, next) => {
+  c.set('hono-context', 'hono-context')
```

Comment:
I need to change the variable name.

Reply:
I think it's better to use the same names with the Pages example: https://github.com/yusukebe/hono-remix-adapter/pull/24/files#diff-1a16d957977e7aaf27b0228dd216f0d8e30696f730d89b6d0d03c1c45fc3babbR14